<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_smene</class>
 <widget class="QMainWindow" name="add_smene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление смен и назачение сотрудников</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../static/resource.qrc">
    <normaloff>:/img/img/Logotip.png</normaloff>:/img/img/Logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>361</width>
       <height>481</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Введите окончание смены</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_save_smene">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back_smene">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
      </property>
      <property name="text">
       <string>Назад к главной панели &quot;Смены&quot;</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_empl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QDateTimeEdit" name="dt_date">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dt_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="displayFormat">
      <string>H:mm</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dt_end">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="displayFormat">
      <string>H:mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Введите дату</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Введите начало смены</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>411</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавление смены</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
